packages/data-point/src/reducer-helpers/constant/constant.test.js

```diff
@@ -1,4 +1,29 @@
-const { ReducerConstant } = require("./constant");
+jest.mock("lodash/cloneDeep");
+const mockCloneDeep = require("lodash/cloneDeep");
+
+mockCloneDeep.mockImplementation(jest.requireActual("lodash/cloneDeep"));
```

Comment:
@raingerber this creates a mock around the real implementation, so we can keep the functionality but track the usage of the method

Reply:
ok, maybe combing this with the jest.mock?
```js
jest.mock("lodash/cloneDeep", {
  return jest.requireActual("lodash/cloneDeep");
});
```

or maybe if we import like this we can use a spy `import { cloneDeep } from 'lodash'`?

Reply:
but then we need the require to track the method, so its really one or the other, at the end they have similar results

Reply:
yeah, I just think it's a little confusing to read